common/scripted_effects/cwp_political_setup.txt

```diff
@@ -476,6 +474,7 @@ setup_starting_ideologies = {
 					or = {
 						has_law = law_type:law_executive
 						has_law = law_type:law_parliamentary
+						has_law = law_type:law_council_elective
```

Comment:
only disagree with this bc of corporate council electives where they'd want oligarchy. But maybe that's better for a leader ideology anyway. like enforcing all elite leaders of corporate states to like oligarchy.